be/src/formats/parquet/column_reader.cpp

```diff
@@ -259,8 +261,10 @@ class MapColumnReader : public ColumnReader {
         // check _value_reader
         if (_key_reader != nullptr) {
             _key_reader->get_levels(def_levels, rep_levels, num_levels);
-        } else {
+        } else if (_value_reader != nullptr) {
             _value_reader->get_levels(def_levels, rep_levels, num_levels);
+        } else {
+            DCHECK(false) << "Unreachable!";
```

Comment:
ditto

Reply:
This function is return void

Reply:
then you should change it to return status